src/main/java/creativei/dao/ReminderCustomDao.java

```diff
@@ -3,8 +3,12 @@
 
 import creativei.entity.Reminder;
 
+import java.util.Date;
 import java.util.List;
 
 public interface ReminderCustomDao {
     Reminder save(Reminder reminder);
+    List<Reminder> findRemindersBetweenDates(Date toDate, Date fromDate);
```

Comment:
Use specifications to club multiple requests.

Reply:
Specification used in commit 3fe67e5